packages/invoices/src/InboundShipment/DetailView/SidePanel/AdditionalInfoSection.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { memo } from 'react';
```

Comment:
I also moved the inbound sections in the side panel out. Just copy pasted except for the related docs section